models/Position.ts

```diff
@@ -0,0 +1,17 @@
+import { Position } from "@/queries/conditional-tokens/types";
+
+export class PositionModel {
+  conditionId: string;
+  outcomeIndex: number;
+  outcomes: string[] | undefined | null;
+
+  constructor(position: Position) {
+    this.conditionId = position.conditionIdsStr;
+    this.outcomeIndex = position.indexSets[0];
+    this.outcomes = position.conditions?.[0]?.outcomes;
+  }
+
+  outcomeString(): string {
+    return this.outcomes ? this.outcomes[this.outcomeIndex - 1] : "";
+  }
```

Comment:
This looks like a frontend thing. Doesn't look like a PositionModel problem. I would define `userOutcome` has a property that can be `this.outcomes[this.outcomeIndex - 1]` or `null` that we define in the constructor.

Reply:
return null when no value, and handle that on componente.